bundle/deploy/state.go

```diff
@@ -0,0 +1,128 @@
+package deploy
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/fileset"
+)
+
+const DeploymentStateFileName = "deployment-state.json"
+
+type File struct {
+	Absolute string `json:"absolute"`
+	Relative string `json:"relative"`
```

Comment:
What are absolute and relative paths in this context?

I had assumed we only need the path relative to the file synchronization root (bundle root today).

Reply:
Good point, we indeed don't need to have it in state, removed